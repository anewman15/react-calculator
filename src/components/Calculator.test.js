/*
 eslint-disable no-unused-vars, import/named, import/no-cycle
*/

import React from 'react';
import { act, render } from '@testing-library/react';
import { unmountComponentAtNode } from 'react-dom';
import renderer from 'react-test-renderer';
import pretty from 'pretty';
import Calculator from './Calculator';

let container = null;
beforeEach(() => {
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

describe('Calculator', () => {
  it('should render a display and button panel', () => {
    const tree = renderer
      .create(<Calculator />)
      .toJSON();
    expect(tree)
      .toMatchSnapshot();
  });
});
