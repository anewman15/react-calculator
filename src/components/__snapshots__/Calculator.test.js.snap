// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a display and button panel 1`] = `""`;
